src/plugins/contextMenu.js

```diff
@@ -85,7 +85,7 @@
               entireColumnSelection = [0, selected[1], this.view.wt.wtTable.getRowStrategy().cellCount - 1, selected[1]],
               columnSelected = entireColumnSelection.join(',') == selected.join(',');
 
-            return selected[0] < 0 || this.countRows() >= this.getSettings().maxRows || columnSelected;
+            return selected[0] < 0 || this.countRows() >= this.getSettings().maxRows || (columnSelected && this.countRows() > 1);
```

Comment:
looks good to me, but it would be better if you assigned `this.countRows()` into a variable, so it doesn't trigger twice in the return statement